doc/nrf/glossary.rst

```diff
@@ -442,6 +442,9 @@ Glossary
    Modulation Coding Scheme (MCS)
       An index defined by the Wi-Fi standards where each index specifies a combination of modulation scheme and forward error correction coding rate.
 
+   Monitor
+      A network interface mode that enables the capturing and analysis of all incoming IEEE 802.11 packets.
```

Comment:
```suggestion
      A network interface mode that enables the capture and analysis of all incoming IEEE 802.11 packets.
```

Reply:
Isn't this Wi-Fi specific? Would be good to mention this in the term definition, such as:
```suggestion
      A network interface mode for nRF70 Series devices that enables the capture and analysis of all incoming IEEE 802.11 packets.
```

Reply:
@chiranjeevi2776 -. Please see this suggestion.